src/pages/console/data/data-model.mdx

```diff
@@ -5,6 +5,12 @@ export const meta = {
 
 ## Data modeling example
 
+<Callout warning>
+
+**Studio now supports all Studio GraphQL APIs, with or without DataStore. [Learn more below](#datastore-and-graphql)**
```

Comment:
Should this be a warning? Info might be better here.

Reply:
Done!